proposals/rcp-45-legacy-deprecated-fields-lookups.md

```diff
@@ -0,0 +1,192 @@
+# Legacy/Deprecated Fields and Lookups
+
+| **RCP** | 045 |
+| :--- | :--- |
+| **Version** | **1.0.0** |
+| **Authors** | Paul Hethmon ([AMP Systems](mailto:paul@amptech.us)) |
+| **Specification** | |
+| **Status** | IN PROGRESS |
+| **Date Ratified** | |
+| **Dependencies** | Data Dictionary 1.7+ |
+| **Related Links** | |
+
+<br /><br />
+
+# RESO End User License Agreement (EULA)
+
+This End User License Agreement (the "EULA") is entered into by and between the Real Estate Standards Organization ("RESO") and the person or entity ("End User") that is downloading or otherwise obtaining the product associated with this EULA ("RESO Product"). This EULA governs End Users use of the RESO Product and End User agrees to the terms of this EULA by downloading or otherwise obtaining or using the RESO Product.
+<br /><br />
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+# Table of Contents
+- [Summary of Changes](#summary-of-changes)
+- [Introduction](#introduction)
+- [Section 1: Field](#section-1-field)
```

Comment:
One note here, usually Section 2 is for the spec, and 3 for Certification.

So your sections should be 2.1, 2.2, etc.